src/main/resources/application.yml

```diff
@@ -11,26 +11,38 @@ spring:
   flyway:
     locations: "classpath:db/migrations"
 
-
   jpa:
     hibernate:
       ddl-auto: validate
     properties:
       hibernate:
         dialect: org.hibernate.dialect.PostgreSQL95Dialect
-        #                show_sql: true
-        #                format_sql: true
+#        show_sql: true
+#        format_sql: true
         jdbc:
           lob:
             non_contextual_creation: true
     open-in-view: false
 
+
 server:
   port: 8081
 
+
 open-library:
   api:
     base-url: https://openlibrary.org/api
     paths:
       books:
         search-by-isbn: /books?format=json&jscmd=data&bibkeys=ISBN:{isbn}
+
+
+authentication:
+  jwt:
+    duration: 3600
+    fail_on_invalid: false
+    fail_on_blacklisted: false
+    allow_anonymous: true
+
+    public-key: MIIBIjANBgkqhkiG9w0BAQEFAAOCAQ8AMIIBCgKCAQEA3Qold2jMzk/Ca3yYusjm9hxNp3EVBnzWD8GbTSfNPV+7z/+2f9LRSx2CztMCwgPciKC+7fWHzKcUgidLbPqPpKjqlbJQ0KKp7rPHOTmAaSaVX+OVSi00p7lbU1Hdepjdzt0LTwT3EBHTU0fGghTOQZffVtKzyg4Gth6iKEhIAHQztQ63JI5c6F1o4eGwJ8JrIHcjUewXO8HDBsU+3UGtiw8ixXVZ4sfGMrWkPylAC81gCWFfQhWggGz0O9xMEBD30C1EgBBZEN8C0Wg3EgDItiUrudqvxcgnCa1ssjEgSY+kYgMsDwcqpzO5PiISNd2JIc31miM99XaQkArYIidBTwIDAQAB
```

Comment:
Just to clarify: these are default keys needed by the tests to run. In production this would be overridden with program arguments.